app/backend/gwells/migrations/0009_create_aquifer_groups.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'm making a change on one of my branches that will make this migration completely redundant.